js/blocks/HeapBlocks.js

```diff
@@ -26,16 +26,17 @@ function setupHeapBlocks() {
                 return;
             }
 
-            var data = [];
-            var url = args[1];
-            var name = args[0];
-            var xmlHttp = new XMLHttpRequest();
+            let data = [];
+            let url = args[1];
+            let name = args[0];
+            let oldHeap;
+            let xmlHttp = new XMLHttpRequest();
```

Comment:
Is this redundant with Line 90?

Reply:
@walterbender No it is not redundant, because line 33  and 90 both belong to different functions means different scopes.
I have also try to remove it but there is an error "xmlHttp is not defined"